test/test_grids.jl

```diff
@@ -198,13 +198,13 @@ function test_regular_rectilinear_xnode_ynode_znode_and_spacings(arch, FT)
         @test min_Δy(grid) ≈ FT(π/3)
         @test min_Δz(grid) ≈ FT(π/3)
 
-        @test xspacings(grid, Center()) ≈ FT(π/N)
-        @test yspacings(grid, Center()) ≈ FT(π/N)
-        @test zspacings(grid, Center()) ≈ FT(π/N)
+        @test all(xspacings(grid, Center()) .≈ FT(π/N))
+        @test all(yspacings(grid, Center()) .≈ FT(π/N))
+        @test all(zspacings(grid, Center()) .≈ FT(π/N))
 
-        @test xspacings(grid, Face()) ≈ FT(π/N)
-        @test yspacings(grid, Face()) ≈ FT(π/N)
-        @test zspacings(grid, Face()) ≈ FT(π/N)
+        @test all(xspacings(grid, Face()) .≈ FT(π/N))
+        @test all(yspacings(grid, Face()) .≈ FT(π/N))
+        @test all(zspacings(grid, Face()) .≈ FT(π/N))
```

Comment:
Could also write this (saving some memory)

```suggestion
        @test all(x ≈ FT(π/N) for x in xspacings(grid, Face()))
        @test all(y ≈ FT(π/N) for y in yspacings(grid, Face()))
        @test all(z ≈ FT(π/N) for z in zspacings(grid, Face()))
```